.github/workflows/haskell.yml

```diff
@@ -59,6 +60,7 @@ jobs:
           - 8.6.5+fusion-plugin
           - 8.6.5+streamk
           - 8.4.4+debug
+          - docspec
```

Comment:
Rename this to `8.10.4+docspec` and move this next to other 8.10.4 tests listed above.